resources/assets/js/annotations/ol/MagicWandInteraction.js

```diff
@@ -195,7 +195,8 @@ class MagicWandInteraction extends PointerInteraction {
         if (this.isShowingCross) {
             this.sketchSource.removeFeature(this.sketchFeature);
         } else {
-            this.dispatchEvent({type: 'drawend', feature: this.sketchFeature});
+            this.filterSketchCoordinates();
```

Comment:
The magic wand interaction creates a polygon at the end. Is the filtering really needed here when the polygon is also checked by the PolygonValidator?